examples/chef/chef_util.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2022 Project CHIP Authors
```

Comment:
try to not name things util/helpers/support/core

They end up being a grab-all content of mixed purpuse and generally when you do `import myutil` it tells the reader of the code nothing of why that import is done or what it contains.

Reply:
Thanks! I have removed this file entirely.